cloudflare/resource_cloudflare_access_policy_test.go

```diff
@@ -12,12 +12,7 @@ func TestAccAccessPolicyServiceToken(t *testing.T) {
 	// Temporarily unset CLOUDFLARE_API_TOKEN if it is set as the Access
 	// service does not yet support the API tokens and it results in
 	// misleading state error messages.
-	if os.Getenv("CLOUDFLARE_API_TOKEN") != "" {
-		defer func(apiToken string) {
-			os.Setenv("CLOUDFLARE_API_TOKEN", apiToken)
-		}(os.Getenv("CLOUDFLARE_API_TOKEN"))
-		os.Setenv("CLOUDFLARE_API_TOKEN", "")
-	}
+	unsetApiToken()
```

Comment:
This hasn't worked; it's failing the integration tests.

```
------- Stdout: -------
=== RUN   TestAccAccessPolicyServiceToken
--- FAIL: TestAccAccessPolicyServiceToken (10.18s)
    testing.go:654: Step 0 error: errors during apply:
        
        Error: error creating access service token: error from makeRequest: HTTP status 501: content "{\"success\":false,\"errors\":[{\"code\":10000,\"message\":\"API Tokens are not supported by this API for now\"}]}\n"
        
          on /opt/teamcity-agent/temp/buildTmp/tf-test386041988/main.tf line 8:
          (source code not available)
        
        
FAIL
```

If you don't want to debug this, I'm happy to see this repeated in the tests that need it as it will be removed once the service supports it.

Reply:
Done - reverted.